ibis/tests/strategies.py

```diff
@@ -187,3 +188,6 @@ def memtable(draw, schema=schema(primitive_dtypes)):  # noqa: B008
 
     df = draw(dataframe)
     return ibis.memtable(df)
+
+
+null_values = st.sampled_from([None, pd.NA])
```

Comment:
Should this include `np.nan`?

Reply:
I don't think so, since that implies we're endorsing `np.nan` as NULL value. While it is used that way in pandas, it's not like that in any other system that isn't based on pandas.

I hope that doesn't sound too ideological. I'm trying to avoid making the mistake of supporting `np.nan` as NULL everywhere and setting expectations according.